examples/js/loaders/ColladaLoader.js

```diff
@@ -18,17 +18,24 @@ THREE.ColladaLoader.prototype = {
 	load: function ( url, onLoad, onProgress, onError ) {
 
 		var scope = this;
-
-		var path = THREE.Loader.prototype.extractUrlBase( url );
-
+		
+                var path = scope.path === undefined ? THREE.Loader.prototype.extractUrlBase( url ) : scope.path; 
+		
 		var loader = new THREE.FileLoader( scope.manager );
+		loader.setPath( scope.path );
```

Comment:
This line is not necessary. Everything else is fine.

Reply:
done.